qutebrowser/config/configtypes.py

```diff
@@ -1996,3 +1996,15 @@ def to_py(
             return urlmatch.UrlPattern(value)
         except urlmatch.ParseError as e:
             raise configexc.ValidationError(value, str(e))
+
+
+class StatusbarWidget(String):
+
+    """A Widget for the status bar.
+
+    Allows some predefined widgets and custom text-widgets via text:$CONTENT."""
+
+    def _validate_valid_values(self, value: str) -> None:
```

Comment:
The validation will also need a test in `tests/unit/config/test_configtypes.py`.